tests/integration/test_notes.py

```diff
@@ -28,3 +29,19 @@ def test_it_returns_404_with_invalid_ticket(self, valid_header):
             headers=valid_header,
         )
         assert is_valid(response, 404)  # Bad Request- 'Invalid Ticket'
+
+    def test_it_deletes_one_note(
+        self, valid_header, create_note, create_admin_user, create_ticket
+    ):
+        note = create_note(create_admin_user(), create_ticket())
```

Comment:
I didn't know we had to pass in arguments to create_note. We'll have to get this updated. Issue #639 created to address this.

Reply:
Thanks for updating the factory, can we update this too?